manifests/source/export.pp

```diff
@@ -42,11 +50,28 @@
     $real_lang = $language
   }
 
+  if (! $organization) or ($organization == '') {
+    # an empty string or "undef" was passed. Use the default
+    # organization.
+    $o = $::sentry::organization
```

Comment:
We use `$real_lang` for the `platform` parameter, but `$o` and `$t` for the new params.  Can you adjust the variable names one way or the other for consistency/readability?